src/force_torque_sensor.cpp

```diff
@@ -0,0 +1,92 @@
+#include <force_torque_sensor.h>
+#include <gazebo-7/gazebo/sensors/Sensor.hh>
```

Comment:
Same as above:
Change ```#include <gazebo-7/gazebo/sensors/ForceTorqueSensor.hh>```
into this ```#include <gazebo/sensors/ForceTorqueSensor.hh>```

Because if we hardcode gazebo-7, then we will have problems once we change to another gazebo version in the future.